ivy/functional/backends/numpy/statistical.py

```diff
@@ -20,10 +20,18 @@ def min(
     *,
     axis: Optional[Union[int, Sequence[int]]] = None,
     keepdims: bool = False,
+    initial: Optional[Union[int, float, complex]] = None,
+    where: Optional[np.ndarray] = None,
     out: Optional[np.ndarray] = None,
 ) -> np.ndarray:
+    if where is None:
```

Comment:
Not sure why we're doing this, doesn't `where=True` and `where=None` have the opposite behaviour?
Rest looks good to me, thanks @Aaryan562 😄

Reply:
Yeah right. I dont remember why I did maybe because I though the default value of where taken by numpy is True but its not the case so yeah reverted this change. Thanks

Reply:
awesome, thanks!